src/PeopleSDKAdapter.js

```diff
@@ -184,4 +185,23 @@ export default class PeopleSDKAdapter extends PeopleAdapter {
 
     return this.getPersonObservables[ID];
   }
+
+  /**
+   * Returns an observable that emits people list
+   *
+   * @param {string} query search query
+   * @returns {external:Observable.<Person>} Observable that emits person list based on search query
+   */
+  getPeopleList(query) {
+    return new Observable(async (observer) => {
+      try {
+        const peopleList = await this.datasource.people.list({displayName: query});
+
+        observer.next(peopleList.items);
+        observer.complete();
+      } catch (err) {
+        observer.error(new Error('error in fetching data'));
```

Comment:
the original error `err` should be logged (otherwise it's lost) and the message here should be more specific so that it is clear what operation failed